fonts.css

```diff
@@ -0,0 +1,9 @@
+@import url  ('https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap');
+@import url  ('https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap');
+
+* {  font-family: 'Nanum Gothic', sans-serif;  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/FW/pullRequest?prid=5459839)